src/main/java/nextstep/subway/path/dto/PathResponse.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.subway.path.dto;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import nextstep.subway.station.domain.Station;
+import nextstep.subway.station.dto.StationResponse;
+
+public class PathResponse {
+
+    List<StationResponse> stations;
+    Double distance;
+
+    public PathResponse() {
+    }
+
+    public PathResponse(List<StationResponse> stations, Double distance) {
```

Comment:
```suggestion
    public PathResponse(List<StationResponse> stations, double distance) {
```
request는 not nullable이 보장되지 않아서 boxing 을 쓰는게 나을 수 있지만
response는 개발자가 통제할 수 있는 범위 내에 있습니다.

불필요한 boxing은 가급적 줄여주세요.

Reply:
항상 궁금하던 부분인데 감사합니다!👍